recipes/fontconfig/meson/conanfile.py

```diff
@@ -50,15 +50,15 @@ def layout(self):
 
     def requirements(self):
         self.requires("freetype/2.13.2")
-        self.requires("expat/2.6.0")
+        self.requires("expat/2.6.2")
```

Comment:
It will conflict with the recent Cairo version bumps (#22696), but v2.6.2 fixes a CVE and should be preferred: https://github.com/libexpat/libexpat/blob/R_2_6_2/expat/Changes#L34-L38